src/NUnitFramework/nunitlite.tests/TestUtilities/TestBuilder.cs

```diff
@@ -48,9 +48,7 @@ public static TestSuite MakeFixture(Type type)
 
         public static TestSuite MakeFixture(object fixture)
         {
-            TestSuite suite = MakeFixture(fixture.GetType());
-            suite.Fixture = fixture;
-            return suite;
+            return MakeFixture(fixture.GetType());
```

Comment:
It *looks* like this is happening within `TestBuilder.RunTestSuite`, although I'm potentially misreading what's going on.